grails-app/views/wps/jobComplete.gsp

```diff
@@ -0,0 +1,11 @@
+Hi,
+
+Your IMOS download is ready. You can download it by clicking here: ${jobReportUrl}
```

Comment:
I'm with you on that. I advised against it also in the original design, as you said, due to added complexity with not much added value for the user.